app/view-plants/page.tsx

```diff
@@ -80,8 +118,23 @@ export default function Page() {
           {viewingOption === 'all' &&
             (inAddMode ? (
               <div>
+                <h3>
+                  {selectedPlants.length > 0
+                    ? `${selectedPlants.length} Plants Selected`
+                    : 'Select Plants'}
+                </h3>
                 {plants.map((plant, key) => (
-                  <PlantCard key={key} plant={plant} canSelect={true} />
+                  <InlineDiv
+                    key={key}
+                    onClick={() => handlePlantCardClick(plant)}
+                  >
+                    <PlantCard
+                      key={key}
+                      plant={plant}
+                      canSelect={true}
+                      isSelected={selectedPlants.includes(plant)}
+                    />
+                  </InlineDiv>
                 ))}
                 <div className="footer">
```

Comment:
we should create style in `view-plants/styles.ts` called `Footer` rather than using className here
also, the select button should be at the top in the same row as "My Garden" and "All" (but that can be for the Style PlantCard PR or a future styling PR)